p2p/client.go

```diff
@@ -282,7 +282,7 @@ func validateSignatureResp(
 		return version.IncompatibleVersionError(header.NodeVersion)
 	}
 
-	err := ideth.ValidateCentrifugeIDBytes(resp.Signature.EntityId, receiver)
+	err := ideth.ValidateCentrifugeIDBytes(resp.Signature.SignerId, receiver)
```

Comment:
this function should be moved to `identity` package.

Reply:
`ValidateCentrifugeIDBytes` sounds mouthful. Can we come up with a better name for this?

Reply:
ValidateDIDBytes